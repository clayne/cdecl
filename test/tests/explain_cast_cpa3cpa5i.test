cdecl | | explain (int(*const(*const)[3])[5])x | 0
