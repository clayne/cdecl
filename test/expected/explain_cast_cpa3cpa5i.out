cast x into constant pointer to array 3 of constant pointer to array 5 of int
